api/lib/domain/services/schooling-registrations-xml-service.js

```diff
@@ -1,91 +1,167 @@
-const { DOMParser } = require('xmldom');
-const xmlBufferToString = require('xml-buffer-tostring');
+const { FileValidationError, SameNationalStudentIdInFileError } = require('../errors');
```

Comment:
Dans la description du commit
`Remove DOMparser that allow a lot of memory.` => `Remove DOMparser that **allocate too much** memory.`